notebooks/208-optical-character-recognition/208-optical-character-recognition.ipynb

```diff
@@ -166,49 +166,19 @@
-    "detection_model_info, recognition_model_info = [\n",
-    "    {\n",
-    "        \"name\": \"horizontal-text-detection-0001\",\n",
-    "        \"composite_model_name\": None,\n",
-    "        \"description\": \"Horizontal text detector based on FCOS with light MobileNetV2 backbone\",\n",
-    "        \"framework\": \"dldt\",\n",
-    "        \"license_url\": \"https://raw.githubusercontent.com/openvinotoolkit/open_model_zoo/master/LICENSE\",\n",
-    "        \"precisions\": [\n",
-    "            \"FP16\",\n",
-    "            \"FP16-INT8\",\n",
-    "            \"FP32\"\n",
-    "        ],\n",
-    "        \"quantization_output_precisions\": [],\n",
-    "        \"subdirectory\": \"intel/horizontal-text-detection-0001\",\n",
-    "        \"task_type\": \"detection\"\n",
-    "    },\n",
-    "    {\n",
-    "        \"name\": \"text-recognition-resnet-fc\",\n",
-    "        \"composite_model_name\": None,\n",
-    "        \"description\": \"\\\"text-recognition-resnet-fc\\\" is a simple and preformant scene text recognition model based on ResNet with Fully Connected text recognition head. Source implementation on a PyTorch* framework could be found here <https://github.com/Media-Smart/vedastr>. Model is able to recognize alphanumeric text.\",\n",
-    "        \"framework\": \"pytorch\",\n",
-    "        \"license_url\": \"https://raw.githubusercontent.com/Media-Smart/vedastr/0fd2a0bd7819ae4daa2a161501e9f1c2ac67e96a/LICENSE\",\n",
-    "        \"precisions\": [\n",
-    "            \"FP16\",\n",
-    "            \"FP32\"\n",
-    "        ],\n",
-    "        \"quantization_output_precisions\": [],\n",
-    "        \"subdirectory\": \"public/text-recognition-resnet-fc\",\n",
-    "        \"task_type\": \"optical_character_recognition\"\n",
-    "    }\n",
-    "]\n",
-    "\n",
-    "for model_info in (detection_model_info, recognition_model_info):\n",
-    "    omz_dir = Path(model_info[\"subdirectory\"])\n",
-    "    omz_model_dir = base_model_dir / omz_dir / precision\n",
-    "    print(omz_model_dir) \n",
-    "    for model_file in omz_model_dir.iterdir():\n",
-    "        try:\n",
-    "            shutil.copyfile(model_file, model_dir / model_file.name)\n",
-    "        except FileExistsError:\n",
-    "            pass\n",
+    "# detection_model_info subdirectory\n",
+    "model_dir.mkdir(exist_ok=True)\n",
+    "\n",
+    "# detection_model_info subdirectory and recognition_model_info subdirectory\n",
+    "omz_model_dirs = [base_model_dir / \"public/text-recognition-resnet-fc\" / precision,\n",
+    "                  base_model_dir / \"intel/horizontal-text-detection-0001\" / precision]\n",
+    "\n",
+    "for omz_model_dir in omz_model_dirs:\n",
+    "    print(f'from {omz_model_dir} to {model_dir} copied:')\n",
+    "    for file_path in omz_model_dir.iterdir():\n",
+    "        if file_path.is_file():\n",
+    "            shutil.copy2(file_path, model_dir)\n",
```

Comment:
I know this is not work done in this PR, but I'm not a fan of downloading anything to some external directory like "~/open_model_zoo_models" and then copying it. I think it creates more mess (more directories containing files related to one case) and unnecessary code. Could we fix it for 208 in this PR? Let's download everything to a local "model" dir.

Reply:
yes, done